### PR TITLE
#101331 Zaakdetail sync through plugins

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

File: zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/autoconfigure/ObjectManagementAutoConfiguration.kt [hunk 1]
```diff
@@ -69,6 +70,7 @@ class ObjectManagementAutoConfiguration {
         )
     }
 
+    @Primary
```
Comment (ivo-ritense) [inline zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/autoconfigure/ObjectManagementAutoConfiguration.kt]: I don't like having primary beans in the valtimo because you can't easily override them. Why is this required here?

Comment (Klaas-Ritense) [reply zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/autoconfigure/ObjectManagementAutoConfiguration.kt]: Changed

File: zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/security/config/ObjectManagementHttpSecurityConfigurer.kt [hunk 2]
```diff
@@ -38,6 +38,8 @@ class ObjectManagementHttpSecurityConfigurer : HttpSecurityConfigurer {
                     .requestMatchers(antMatcher(DELETE, "$CONFIGURATION_URL/{id}")).hasAuthority(ADMIN)
                     .requestMatchers(antMatcher(GET, "$CONFIGURATION_URL/{id}/object")).authenticated()
                     .requestMatchers(antMatcher(POST, "$CONFIGURATION_URL/{id}/object")).authenticated()
+
+                    .requestMatchers(antMatcher(GET, CONFIGURATION_MANAGEMENT_URL)).authenticated()
```
Comment (theo-ritense) [inline zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/security/config/ObjectManagementHttpSecurityConfigurer.kt]: This should be ADMIN only

File: zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/client/ObjectenApiClient.kt [hunk 3]
```diff
@@ -260,14 +260,25 @@ class ObjectenApiClient(
         objectUrl: URI,
         objectRequest: ObjectRequest
     ): ObjectWrapper {
+        val objectRequestCorrectedHost = if (objectRequest.type.host == "localhost") {
```
Comment (theo-ritense) [inline zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/client/ObjectenApiClient.kt:263]: We should make a filter for this that is only applied when running in development

File: zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/security/config/ObjectManagementHttpSecurityConfigurer.kt (hunk 2 above)
Comment (Klaas-Ritense) [reply zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/security/config/ObjectManagementHttpSecurityConfigurer.kt]: Fixed

**Review by theo-ritense — APPROVED** [review]
